README.md

```diff
@@ -49,6 +50,68 @@ try {
 
     // From a Uint8Array with a lenght of 32
     Record::fromUint8Array([1, 2, 3, 255]);
+
+    // From a JSON Document
+    Record::fromJSON({ 'hello': 'world' })
+} catch (Exception $e) {
+    // Handle exception
+}
+```
+
+### Sign Record
+You can sign a Record if it was constructed from JSON file. To use the sign functionality you need to provide private key as it's going to be used for the digital signature process. Remember to keep it safe!
+
+**How can I generate a private key?**
+
+Bloock libraries uses the ES256K signature algorithm that uses the secp256k1 <http://www.secg.org/sec2-v2.pdf> elliptic curve.
+
+The private key needed to use the signing functionality needs to be specific for the elliptic curve used.
+
+There are two ways to generate one:
+1. Use a secp256k1 library (<https://github.com/simplito/elliptic-php>)
+2. (Not recommended) Use an online tool to generate one, such as <https://secretscan.org/generator>.
+
+This example shows how to sign records:
+```php
+<?php
+
+use Bloock\Record\Entity\Record;
+
+require_once(__DIR__ . "/vendor/autoload.php");
+
+try {
+    $json = {'hello': 'world'};
```

Comment:
This isn't PHP syntax and will not work. Please, review in a separate script that each code snippet works as expected.